src/Backends/Helmholtz/VLERoutines.cpp

```diff
@@ -1173,7 +1173,18 @@ void SaturationSolvers::successive_substitution(HelmholtzEOSMixtureBackend &HEOS
             df += dfdK*(deriv_liq-deriv_vap);
         }
         
-        change = -f/df;
+        if (std::abs(df) <= 1e-12) { // To avoid dividing by 0
+            if (std::abs(f) <= 1e-12)
++            {
++                change = -f; // Should be converged. f <= e-12, so change will have nearly no impact.
```

Comment:
The loop tolerance being hard coded as [std::abs(f) > 1e-12](https://github.com/JonWel/CoolProp-f/blob/patch-4/src/Backends/Helmholtz/VLERoutines.cpp#L1212) this will just finish the loop and exit.
I thought about putting a break, but in that case, the updated K would not be used, so the update just after the loop be just useless (even if K should not be extremely different than the previous one).